services/static-webserver/client/source/boot/index.html

```diff
@@ -56,4 +56,12 @@
   ${preBootJs}
   <script type="text/javascript" src="${appPath}boot.js"></script>
 </body>
+<script>
+  window.markerConfig = {
+    project: '655285eac2fdc551da905141',
+    source: 'snippet'
+  };
+
```

Comment:
IMO this should only be available in the s4l products

Reply:
Did you get an account?

Reply:
I created a test account and just want to see how it works. We well configure it properly (only visible after login, only for sim4life.io) later.